timely_beliefs/beliefs/utils.py

```diff
@@ -149,6 +149,20 @@ def respect_event_resolution(grouper: DataFrameGroupBy, resolution):
     return df
 
 
+def propagate_beliefs(
+    df: "classes.BeliefsDataFrame",
+) -> "classes.BeliefsDataFrame":
+    """Propagate beliefs over time.
```

Comment:
Sad that your example from the PR (or a smaller version of it) isn't here. That would help me in the future ― what does propagating really mean?